source/stylesheets/mixins/_media_queries.scss

```diff
@@ -2,7 +2,7 @@
 $breakpoints: (
   phone: 600px,
   tablet: 768px,
-  desktop: 1150px,
+  desktop: 1205px,
```

Comment:
To account for the new padding on the screening section. Since we are positioning the video using `absolute`, there's padding on the left but not on the right of the section. As a result, the video started to break too late.